uploader/api_commands.py

```diff
@@ -1,65 +1,62 @@
 import datetime
 import os
-
+from celery import Celery
 from flask import request
-from moviepy.editor import VideoFileClip, concatenate_videoclips, ImageClip
 from werkzeug.utils import secure_filename
-
 from base import app
 
+celery_app = Celery("videos", broker='redis://broker:6379/0')
+
+
+@celery_app.task(name="procesar_video")
+def procesar_video(*args):
+    pass
+
```

Comment:
Es posible mover esta parte del codigo al base.py?

Reply:
Hmm... me parece raro. Creo que sólo este endpoint debería estar acoplado a celery.

Reply:
Puedes mover solamente celery_app = Celery("videos", broker='redis://broker:6379/0') e importar la variable celery_app

Reply:
should be fixed now. hice lo que me recomendó juan. muchas gracias